docs/modules/ROOT/pages/reference/extensions/openapi-java.adoc

```diff
@@ -39,6 +39,26 @@ Or add the coordinates to your existing project:
 
 Check the xref:user-guide/index.adoc[User guide] for more information about writing Camel Quarkus applications.
 
+== Usage
+
+You can use this extension to expose REST DSL services to quarkus openapi. With
+the `quarkus-smallrye-openapi`, you can access them by `/q/openapi?format=json`.
+
+Refer to the https://quarkus.io/guides/openapi-swaggerui[Quarkus OpenAPI guide] for further information.
+
+By default, this experimental feature disable. You can enable it by
+[source, properties]
+----
+quarkus.camel.openapi.expose.enabled=true
+----
+
+[WARNING]
+====
+It doesn't support the rests services used in `org.apache.camel.builder.LambdaRouteBuilder` right now.
+Also, it can not use CDI injection in the RouteBuilder `configure()` since we get the rest definitions at build time while the CDI is unavailable.
```

Comment:
```suggestion
Also, it can not use CDI injection in the RouteBuilder `configure()` since we get the rest definitions at build time while  CDI is unavailable.
```